ios-sdk/src/MapViewController.swift

```diff
@@ -46,4 +48,23 @@ public class MapViewController: TGMapViewController {
         }
         return true
     }
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        LocationManager.sharedManager.delegate = self
+    }
+
+    //MARK: - LocationManagerDelegate
+
+    public func locationDidUpdate(location: CLLocation) {
+        guard let marker = currentLocationGem else {
+            return
+        }
+        lastSetPoint = TGGeoPoint(longitude: location.coordinate.longitude, latitude: location.coordinate.latitude)
+        markerSetPoint(marker, coordinates: TGGeoPoint(longitude: location.coordinate.longitude, latitude: location.coordinate.latitude))
```

Comment:
You can but this is a more succinct API because `lastSetPoint` is an optional, and so i'd have to unwrap `lastSetPoint` in some fashion, either using `if let` / `guard let` or force unwrapping. It's a bit of an annoyance with swift.